pkg/deploy/controller/configchange/controller.go

```diff
@@ -52,6 +55,7 @@ func (c *DeploymentConfigChangeController) Handle(config *deployapi.DeploymentCo
 			if kerrors.IsConflict(err) {
 				return fatalError(fmt.Sprintf("DeploymentConfig %s updated since retrieval; aborting trigger: %v", deployutil.LabelForDeploymentConfig(config), err))
 			}
+			c.recorder.Eventf(config, "failedCreate", "Couldn't create initial deployment: %v", err)
```

Comment:
Not at the moment. It'll only happen when more DC events roll in. The reflector for this controller only resyncs every few minutes. The configChange controller currently only retries once until #3051 is resolved (at which point this should be rate limited/time based).